cosmos/dbt/graph.py

```diff
@@ -215,24 +209,30 @@ def load_from_dbt_manifest(self):
         * self.nodes
         * self.filtered_nodes
         """
+        if not self.project_config.is_manifest_available():
+            raise CosmosValueError(f"Unable to load manifest using {self.project_config.manifest_path}")
+
         logger.info("Trying to parse the dbt project using a dbt manifest...")
         nodes = {}
-        with open(self.project.manifest_path) as fp:
-            manifest = json.load(fp)
+        with self.project_config.manifest_path.open() as manifest_file:
```

Comment:
This is less readable than what we have before, I suggest to what we had before.

Reply:
this felt like the more pathlib-native way of doing it. I don't think there's a reason to change it back unless you feel very strongly.